src/lib/fs-utils.ts

```diff
@@ -1,21 +1,25 @@
 import * as Bluebird from 'bluebird';
 import * as _ from 'lodash';
-import { fs } from 'mz';
+import { promises as fs } from 'fs';
 import * as Path from 'path';
-import * as constants from './constants';
+import { exec } from 'child_process';
+import { promisify } from 'util';
+
 import { ENOENT } from './errors';
 
+export const execAsync = promisify(exec);
+
 export function writeAndSyncFile(
 	path: string,
 	data: string | Buffer,
 ): Bluebird<void> {
-	return Bluebird.resolve(fs.open(path, 'w')).then((fd) => {
+	return Bluebird.resolve(fs.open(path, 'w')).then((fh) => {
```

Comment:
Fwiw I normally see `fd` used as in file descriptor and so I'd stick with `fd` as it's what I've always seen to name a "file descriptor" and makes it easier for me when reading the code (I know it is technically a file handle, as in a data structure that contains a file descriptor, but I've never seen people bother to differentiate at the var name level before)

Reply:
I would keep it as is normally, but this change was made to reduce confusion, as VSCode Intellisense was showing  `fd` as type fs.FileHandle`. So fh seemed more appropriate.

Reply:
I agree with Page but I suppose fh is the most correct so...I'm for `fh`

Reply:
I'm for `fd`. Whenever I see `fd` in a piece of code I know that code is dealing with a file (for me it comes from writing C), which makes it more readable at least for me.

Reply:
It seems we have a 50/50 split. With regards with `fh` vs `fd` being more familiar to devs, from my experience with Python, I am more familiar with `filehandle`. You can really make the argument for either one based on past experience, but we are writing a TypeScript module. Compromise: for *absolute* clarity how about I rename this variable to `file`?